evaluate/repository_test.go

```diff
@@ -0,0 +1,68 @@
+package evaluate
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+	"github.com/symflower/eval-dev-quality/language"
+	"github.com/symflower/eval-dev-quality/model"
+	"github.com/symflower/eval-dev-quality/model/symflower"
+	"github.com/zimmski/osutil"
```

Comment:
split imports

Reply:
Yarg. Time to get the Symflower-linter into the symflower binary so we can use it here.